media-sound/wav2json/wav2json-0.4.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+COMMIT_ID="e412923f1f792408e8ae1096ca40fb5307ddfc74"
+DESCRIPTION="Generate waveformjs.org compatible json data out of wav files"
+HOMEPAGE="https://github.com/beschulz/wav2json"
+SRC_URI="https://github.com/beschulz/wav2json/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
+
+S=${WORKDIR}/${PN}-${COMMIT_ID}/build
+
+LICENSE="GPL-1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	dev-libs/boost
+	media-libs/libsndfile
+"
+RDEPEND="${DEPEND}"
+
+src_install() {
+	dobin ../bin/Linux/wav2json
```

Comment:
are you sure it doesn't install any other stuff with `make install` or something? Just asking, haven't checked the BUILD_DIR myself